tagread_mqtt.yaml

```diff
@@ -0,0 +1,105 @@
+# Insert your SSID and Your PWD after inital setup
+wifi:
+# networks:
+#    - ssid: 'REPLACEME'          # Enter your WiFi SSID here. Example: `ssid: 'your_network_name'`
+#      password: 'REPLACEME'      # Enter your wifi password here. Example: `password: 'abcde123456'`
+  ap:
+    ssid: ${name}
```

Comment:
Variable name shall be "devicename", isn't it?

Reply:
indeed